src/components/Book.js

```diff
@@ -1,16 +1,46 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 
 const Book = ({ book, handleRemoveBook }) => {
   const handleBookRemove = () => {
     handleRemoveBook(book);
   };
+  const percentage = 66;
   return (
-    <tr key={book.id} id={book.id}>
-      <td>{book.title}</td>
-      <td>{book.category}</td>
-      <td><button type="button" onClick={handleBookRemove} className="btn btn-danger">Delete</button></td>
-    </tr>
+    <div className="card">
+      <div className="row card-inside">
+        <div className="col-12 col-sm-3 col-md-4">
+          <p className="gender-book">{book.category}</p>
+          <p className="title-book">{book.title}</p>
+          <p className="author-book">JRR Tolkien</p>
+          <div className="row">
+            <div className="col-12 col-lg-5">
+              <a className="book-option" href="/">Comments</a>
+            </div>
+            <div className="col-12 col-lg-4">
+              <a className="book-option" href="/">Remove</a>
+            </div>
+            <div className="col-12 col-lg-3">
+              <a className="book-option" href="/">Edit</a>
```

Comment:
### [OPTIONAL]

- [ ] The options are anchor tags leading to a root path that is causing the page to reload if clicked, either make them lead to no path by adding '#' or using a different tag to represent them.

Reply:
- At the first time I tried to use `#` but I couldn't, the linters rules doesn't approve it.